guides/common/modules/proc_creating-a-location-by-using-cli.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="Creating_a_Location_{context}-by-using-cli"]
+= Creating a location by using Hammer CLI
+
+[role="_abstract"]
+Use this procedure to create a location so that you can manage your hosts and resources by location by using Hammer CLI.
+
+.Procedure
+* Create a location:
++
+[subs="+quotes"]
+----
+$ hammer location create \
+--description "_My_Location_Description_" \
+--name "_My_Location_" \
+--parent-id "_My_Location_Parent_ID_"
+----
+
+.Creating multiple locations
```

Comment:
This is a procedure from elsewhere, I turned it into an example here. The content is not new.